# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1465832119 14400
#      Mon Jun 13 11:35:19 2016 -0400
# Node ID 525b46f1d48ff48e11d684e93b3dd5bf43351849
# Parent  4c0c6f48bdc7bcda7bbe27c34ca7ff10f285a9ef
* kpse.cc (find_first_of): Use iostream instead of stdio for debug message.

diff --git a/liboctave/util/kpse.cc b/liboctave/util/kpse.cc
--- a/liboctave/util/kpse.cc
+++ b/liboctave/util/kpse.cc
@@ -625,17 +625,17 @@ find_first_of (const std::string& path, 
             }
 
           std::cerr << ") =>";
         }
 
       log_search (ret_list);
 
       if (KPSE_DEBUG_P (KPSE_DEBUG_SEARCH))
-        gnulib::putc ('\n', stderr);
+        std::cerr << std::endl;
     }
 
   return ret_list;
 }
 
 /* Search each element of PATH for each element of NAMES.  Return the
    first one found.  */
 
